cli/cmd/check.go

```diff
@@ -222,7 +224,18 @@ func configureAndRunChecks(cmd *cobra.Command, wout io.Writer, werr io.Writer, s
 		os.Exit(1)
 	}
 
-	if !success || !extensionSuccess {
+	totalSuccess := success && extensionSuccess
+
+	if options.output == healthcheck.ExtensionShortOutput {
```

Comment:
After removing `ExtensionShortOutput`

```suggestion
	if options.output == healthcheck.ShortOutput {
```